pkg/webui/console/components/rights-group/rights-group.styl

```
@@ -22,6 +22,10 @@
 .right-label
   span::first-letter
     text-transform: uppercase
+  flex-wrap: wrap
 
 .grant-type
   margin-bottom: $cs.s
+
+.description
+  color: $tc-subtle-gray
```

```suggestion
  color: $tc-subtle-gray
  width: 100%
```
Otherwise, short descriptions will be rendered on the same line.